.github/ISSUE_TEMPLATE.md

```diff
@@ -2,34 +2,34 @@
 <!--- Also test if the latest release and devel branch are affected too -->
```

Comment:
It might be not relevant to the current PR, but the correct comment start is `<!--`. Was this an intentional change to triple dashes?

Reply:
Yes, this is intentional. The idea of a triple dash is that these comments to not leak into the eventual rendered document. But at this time GitHub does not support it.

The result would be that the comments would only be there the first edit, and disappear subsequent edits.

Reply:
But it also stands out a bit more ;-)